### PR TITLE
Style Filterable Table

### PR DESCRIPTION
**Pull request by TheGostKasper**
**Regarding [#2477
](https://github.com/weaveworks/weave-gitops/issues/2477)**

##### This PR only fix :
- Scroll-able `overflow` for `ChipGroup `
- Add bg for Table header 

--------------------------------
**Bug**
image

**Fix**
image

### CODE REVIEW

**Review by jpellizzari — APPROVED** [review]

**Comment by JamWils** [comment]
What issue is this actually tied to?

**Comment by TheGostKasper** [comment]
@JamWils i added the issue to the description

**Comment by joshri** [comment]
Really, really cool work on the scrollbar. Very nice. LOVE IT. 🌞

File: ui/lib/theme.ts [hunk 1]
```diff
@@ -96,6 +97,26 @@ export const GlobalStyle = createGlobalStyle`
   .auth-modal-size {
     min-height: 475px
   }
+  body::-webkit-scrollbar-track, .fancy-scroll::-webkit-scrollbar-track {
```
Comment (joshri) [inline ui/lib/theme.ts:100]: This is so good that I'm wondering if we even want to limit it to a class name. Why not override every ugly windows scrollbar? But that's a question for another day I think. Really nice work

**Review by joshri — APPROVED** [review]

**Pull request merged**